common/k8s/src/middleware/metadata.rs

```diff
@@ -225,13 +225,15 @@ impl K8sMetadata {
         let store_writer = reflector::store::Writer::default();
         let store = store_writer.as_reader();
 
-        let params = if let Some(node) = node_name {
-            ListParams::default().fields(&format!("spec.nodeName={}", node))
+        let wc = WatcherConfig::default().timeout(30).any_semantic();
```

Comment:
Could this potentially be a fix for the issue we've seen where in some cases the watcher becomes un-responsive after the API returns an error? https://github.com/logdna/logdna-agent-v2/pull/491

Reply:
Right now our fix is a bit ugly, requiring us to recreate the watcher on errors

Reply:
also 30 - make it configurable and/or name this default val as const?